samples/openapi3/client/petstore/python-experimental/petstore_api/model/nullable_shape.py

```diff
@@ -120,5 +121,6 @@ def __new__(
             **kwargs,
         )
 
+from petstore_api.model.errorunknown import ERRORUNKNOWN
```

Comment:
@spacether these changes break the build. I wonder if you can take a look when you've time. Thanks.

Reply:
I fixed this